### PR TITLE
Dev/logloss

### PR DESCRIPTION
**Pull request by avolkov-intel**
# Description
Please include a summary of the change. For large or complex changes please include enough information to introduce your change and explain motivation for it.

Changes proposed in this pull request:
-
-
-

### CODE REVIEW

File: cpp/oneapi/dal/backend/primitives/objective_function/test/logloss_dpc.cpp [hunk 1]
```diff
@@ -0,0 +1,499 @@
...
+                              const float_t L2,
+                              const float_t rtol = 1e-3,
+                              const float_t atol = 1e-3) {
+        const std::int64_t p = data.get_dimension(1);
+        auto out_hessian =
+            ndarray<double, 2>::empty(this->get_queue(), { p + 1, p + 1 }, sycl::usm::alloc::host);
+
+        naive_hessian(data, probabilities, out_hessian, L2);
+
+        for (std::int64_t i = 0; i <= p; ++i) {
```
Comment (KulikovNikita) [inline cpp/oneapi/dal/backend/primitives/objective_function/test/logloss_dpc.cpp:353]:
```suggestion
        for (std::int64_t i = 0; i < (p + 1); ++i) {
```

**Review by KulikovNikita — APPROVED** [review]

**Comment by avolkov-intel** [comment]
http://intel-ci.intel.com/edb849db-6891-f1c6-9df5-a4bf010d0e2e

**Pull request merged**